### ISSUE TITLE
[Bug]: PlayerInteractEvent 

### ISSUE THREAD

**Issue by Zarkness2** [CoderKuo/TrMenu#38]
### 🚀 Have you tried the latest version

- [X] I have updated TrMenu to the latest version and the same bug is reproduced.

### 📚 Have you checked the Wiki

- [X] I have checked the Wiki's related content about the problem and determined that the problem exists.

### 🔗 Is there a duplicate Issue

- [X] I have looked through the existing issue and found no duplicate content, or a more detailed description of the existing issue.

### 📰 Server details

```
TrMenu Dump Information (Date: 2024-02-26 09:07:12)

| Server OS: Windows 11 amd64 10.0
| Server software: git-Paper-430 (MC: 1.20.4) (1.20.4-R0.1-SNAPSHOT)
| Java version: 17.0.9

| TrMenu: 3.2.3-SNAPSHOT
Installed Plugins: 
· AfkPool - 1.3.1 (0 KB)
· BKCommonLib - 1.20.4-v2 (0 KB)
· BetterGUI - 9.4 (0 KB)
· BlockEffects - 3.0.1 (0 KB)
· CoinsEngine - 2.2.0 (0 KB)
· ConditionalEvents - 4.45.5 (0 KB)
· CustomJoinMessages - 17.3.1 (0 KB)
· DecentHolograms - 2.8.6 (0 KB)
· DeluxeMentions - 5.9.20 (0 KB)
· EnhancedGlistBridge - 2.0.0-BETA7-SNAPSHOT (0 KB)
· EternalTags - 1.2.4 (0 KB)
· FastAsyncWorldEdit - 2.8.5-SNAPSHOT-648;9ffe76d (0 KB)
· FlightControl - 5.0.0-BETA (0 KB)
· GravityControl - 2.0.0 (0 KB)
· HeadBlocks - 2.4.5 (0 KB)
· HeadDatabase - 4.18.4 (0 KB)
· ItemJoin - 6.0.6-SNAPSHOT-b${env.BUILD_NUMBER} (0 KB)
· ItsMyConfig - 2.2.2 (0 KB)
· JukeBox - 1.20.9 (0 KB)
· LibertyBans - 1.1.0-SNAPSHOT-2024-02-16T15:06:43Z (0 KB)
· LibsDisguises - 10.0.42-SNAPSHOT (0 KB)
· LuckPerms - 5.4.117 (0 KB)
· Maintenance - 4.2.1-SNAPSHOT (0 KB)
· MaintenanceAddon - 2.0.5 (0 KB)
· MyCommand - 5.7.4 (0 KB)
· My_Worlds - 1.20.4-v3 (0 KB)
· NoRain - 1.4.1 (0 KB)
· NoteBlockAPI - 1.6.2 (0 KB)
· Pl-Hide-Pro - 2.8.2 (0 KB)
· PlaceholderAPI - 2.11.5 (0 KB)
· PlayerHide - 1.3.1 (0 KB)
· PlayerParticles - 8.4 (0 KB)
· PremiumVanish - 2.9.5 (0 KB)
· ProtocolLib - 5.2.0-SNAPSHOT-679 (0 KB)
· RoseChat - Beta-20 (0 KB)
· SetSpawn - 4.8 (0 KB)
· SkinsRestorer - 15.0.7 (0 KB)
· TAB - 4.1.3-SNAPSHOT (0 KB)
· TrMenu - 3.2.3-SNAPSHOT (0 KB)
· UltraCosmetics - 3.4.2 (0 KB)
· VoidTeleportPlus - 3.5.1 (0 KB)
· WorldGuard - 7.0.9+5934e49 (0 KB)
· ZNPCsPlus - 2.0.0-SNAPSHOT (0 KB)
· ajParkour - 2.12.10 (0 KB)
· eGlow - 3.2.8 (0 KB)
· nightcore - 2.5.1 (0 KB)
· spark - 1.10.59 (0 KB)
· zMenu - 1.0.2.6 (0 KB)
```

### 🔍 Detailed description

the error occurs every time I run a menu command from a head with itemjoin texture, not only with menus that I make but also with the default menus.

### 🗄 Server log

```
[09:02:17 ERROR]: Could not pass event PlayerInteractEvent to TrMenu v3.2.3-SNAPSHOT
java.lang.NoSuchMethodError: 'java.lang.String com.mojang.authlib.properties.Property.getName()'
	at me.arasple.mc.trmenu.util.bukkit.Heads.seekTexture(Heads.kt:73) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.util.bukkit.ItemMatcher$Match$itemsMatcher$1.invoke(ItemMatcher.kt:143) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.util.bukkit.ItemMatcher$Match$itemsMatcher$1.invoke(ItemMatcher.kt:122) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.util.bukkit.ItemMatcher.itemMatches(ItemMatcher.kt:53) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.module.internal.listener.ListenerItemInteract.onInteract(ListenerItemInteract.kt:38) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at java.lang.invoke.MethodHandle.invokeWithArguments(Unknown Source) ~[?:?]
	at me.arasple.mc.trmenu.taboolib.library.reflex.JavaClassMethod.invoke(JavaClassMethod.kt:31) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.event.EventBus.invoke(EventBus.kt:132) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.event.EventBus.invoke$default(EventBus.kt:130) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.event.EventBus$registerBukkit$2.invoke(EventBus.kt:77) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.event.EventBus$registerBukkit$2.invoke(EventBus.kt:76) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.function.ListenerKt$registerBukkitListener$1.invoke(Listener.kt:39) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.common.platform.function.ListenerKt$registerBukkitListener$1.invoke(Listener.kt:39) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.platform.BukkitListener$registerListener$listener$1.invoke(BukkitListener.kt:36) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.platform.BukkitListener$registerListener$listener$1.invoke(BukkitListener.kt:36) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at me.arasple.mc.trmenu.taboolib.platform.BukkitListener$BukkitListener.execute(BukkitListener.kt:73) ~[TrMenu-3.2.3-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:git-Paper-430]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.20.4.jar:git-Paper-430]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.20.4.jar:git-Paper-430]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R3.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:592) ~[paper-1.20.4.jar:git-Paper-430]
	at org.bukkit.craftbukkit.v1_20_R3.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:548) ~[paper-1.20.4.jar:git-Paper-430]
	at org.bukkit.craftbukkit.v1_20_R3.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:543) ~[paper-1.20.4.jar:git-Paper-430]
	at org.bukkit.craftbukkit.v1_20_R3.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:539) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:1993) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:32) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.a(ServerboundUseItemPacket.java:8) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:54) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1455) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:194) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1432) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1355) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1333) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1222) ~[paper-1.20.4.jar:git-Paper-430]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-430]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
```

### ⚙ Configurations

```

```

**Comment by Dreeam-qwq** [comment]
Fixed in 6baa0a064625f0fc26476c86ff8fd6223ccb0482

**Comment by Zarkness2** [comment]
thx
